code/controllers/Processes/mob.dm

```diff
@@ -14,7 +14,7 @@
 /datum/controller/process/mob/doWork()
 	for(last_object in mob_list)
 		var/mob/M = last_object
-		if(isnull(M.gcDestroyed))
+		if(M && isnull(M.gcDestroyed))
```

Comment:
this change to the mob controller was added after discussion with skull. Bees managed to turn up an unusual edge case bug and this was the best means of preventing it